### PR TITLE
feat: rabbitmq, nats and rest as relay services

### LINKED ISSUE #120: Move comms with RabbitMQ, REST to startup-lib
https://github.com/tazama-lf/relay-service/issues/8

### PR DESCRIPTION
**Pull request by rtkay123**
## What did we change?
Added relay services

## How was it tested?
- [x] Locally
- [x] Development Environment
- [ ] Not needed, changes very basic
- [x] Husky successfully run
- [ ] Unit tests passing and Documentation done

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,9 @@
   },
   "dependencies": {
     "@tazama-lf/frms-coe-lib": "^5.0.0-rc.4",
+    "@types/amqplib": "^0.10.5",
```
Comment (cshezi) [inline package.json]:
```suggestion
```

Comment (cshezi) [reply package.json]: Consider adding this type definer lib in the dev deps

File: src/services/natsRelayService.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { type ILoggerService } from '../interfaces';
+import { startupConfig } from '../interfaces/iStartupConfig';
+
+export class NatsRelay implements IRelay {
+  private readonly config = relayConfig;
+  private NatsConn_Producer?: NatsConnection;
+  private logger?: ILoggerService | Console;
+
+  async init(loggerService?: ILoggerService): Promise<void> {
+    if (loggerService) {
```
Comment (cshezi) [inline src/services/natsRelayService.ts]: Suggestion: Create a function for your logger client decision-making
Reason: Code reuse 
E.g Change dev to development would need 3 lines changed

**Review by cshezi — APPROVED** [review]
💯 LGTM

**Review by Sandy-at-Tazama — APPROVED** [review]

**Pull request merged**